app/controllers/reports_controller.rb

```diff
@@ -1,21 +1,27 @@
 # frozen_string_literal: true
 
+require 'datatables/report_datatable'
+
 class ReportsController < ApplicationController
   before_action :set_report, only: %i[show edit update destroy]
 
   def index; end
 
   def show; end
 
-  def dttb_index
-    authorize @reports = Report.includes(:reportable, :technology, :user).order(date: :desc)
-
-    respond_to do |format|
-      format.html
-      format.json { render 'index.json' }
-    end
+  def datatable
```

Comment:
The searchbox in the view is not working. Any ideas? I permitted all the params. It's not returning an error and does seem to be performing a query, just not actually filtering results as expected. I don't see anything on the gem's Github page about searching.

Reply:
Oh, no. That is disappointing!  There must be something else on my system that I didn't convey in this PR. Let me try to figure what that is.

Reply:
Ok, I found a missing comma; fixed that. I ran the server with _RAILS_ENV=production_ which did not provide a search box. I was able to correct this with _bundle exec rake assets:precompile_. And I removed a few unnecessary _puts_. Unpermitted params are not breaking search in development nor when I ran _RAILS_ENV=production_.

![polymorphic_search](https://user-images.githubusercontent.com/54211615/110038868-5c827600-7d0e-11eb-92d1-1014151bc0da.gif)

Reply:
@chiperific If it still isn't working for you, we should arrange a zoom call or something to troubleshoot.

Reply:
@chiperific  I was able to resolve the strong parameters issue -- finally. I am glad I get to work on this project as I this has been my only exposure to SPs. I am not sure how they are handled within the monolith I work on -- I need to find out. I am skipping SP for the datatable controller only. I prefer to POST the datable params instead of GET, which is what led to the SP issue. If running _bundle exec rake assets:precompile_ and these recent commits don't have it working right away for you, please let me know so we can jump on a call together and get it working.

Reply:
It works!! And I didn't need to precompile the assets!